gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GrpcRequestTracingInfo.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.hadoop.gcsio;
+
+import java.util.OptionalLong;
+
+public class GrpcRequestTracingInfo {
```

Comment:
Make this class package-private and add javadoc

Reply:
Opened b/239642937 to track this.